data/items.ts

```diff
@@ -6677,6 +6677,12 @@ export const Items: {[itemid: string]: ItemData} = {
 			},
 		},
 		onUpdate(pokemon) {
+			const itemUpdateIgnoredAbilities = [('grimneigh' as ID), ('beastboost' as ID), ('soulheart' as ID), ('chillingneigh' as ID), ('moxie' as ID)];
```

Comment:
I worry that this will cause other legitimate interactions, e.g. a Beast Boost Pokemon attacking into Gooey and KOing it, from resolving the Speed drop.

Reply:
It probably will, yeah. I'll have to look into beast boost on it's own to provide a better solution, and I'll have to look out for edge cases like this.

Just to confirm, the resolve order should be Attack into Gooey -> Kill -> Move Stat Drop (if applicable, i.e. Superpower) -> Speed Drop -> Ability Boost -> White Herb?

Reply:
Tested Beast Boost into Gooey and KO, all stat drops are resolved.

Current Order is:

Kill -> Move Stat Drop -> Gooey Speed Drop -> Ability Boost -> White Herb

If this order is not intended, let me know and I'll see what I can do to have this order swapped around as necessary.